lib/omnibus-ctl.rb

```diff
@@ -394,6 +398,16 @@ def show_config(*args)
       end
     end
 
+    def show_versions(*args)
+      f = File.open("#{base_path}/version-manifest.txt")
+
+      while line = f.gets do
```

Comment:
A filter of results using a search term of any remaining args would be nice.   Would also need to change arity -> 2.